orchard-core/src/main/scala/com/salesforce/mce/orchard/system/OrchardSystem.scala

```diff
@@ -32,7 +32,7 @@ object OrchardSystem {
   private case object HeartBeat extends Msg
   private case object AdoptOrphanWorkflows extends Msg
 
-  def apply(database: OrchardDatabase): Behavior[Msg] = Behaviors.setup { ctx =>
+  def apply(database: OrchardDatabase, checkProgressDelay: FiniteDuration): Behavior[Msg] = Behaviors.setup { ctx =>
```

Comment:
change this to to take `OrchardSettings` instead, so that we can add more configurations later